tests/Database/DatabaseFunctionalTest.php

```diff
@@ -23,18 +24,18 @@ class DatabaseFunctionalTest extends FunctionalTestCase
     /**
      * @dataProvider provideInvalidDatabaseNames
      */
-    public function testConstructorDatabaseNameArgument($databaseName): void
+    public function testConstructorDatabaseNameArgument($databaseName, string $expectedExceptionClass): void
     {
-        $this->expectException(InvalidArgumentException::class);
+        $this->expectException($expectedExceptionClass);
         // TODO: Move to unit test once ManagerInterface can be mocked (PHPC-378)
         new Database($this->manager, $databaseName);
     }
 
     public function provideInvalidDatabaseNames()
     {
         return [
-            [null],
-            [''],
+            [null, TypeError::class],
+            ['', InvalidArgumentException::class],
```

Comment:
Would it make sense to move these `provideInvalidDatabaseNames` and `provideInvalidCollectionNames` data providers up to a base test class? That would remove the need to redefine them in all of these files. Could also punt this to a future code cleanup ticket (great for onboarding), as it's not really relevant to this PR.

Reply:
Disregard, I misunderstood how these were being used.